flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/dql/SqlRichDescribeTable.java

```diff
@@ -39,7 +39,7 @@ public class SqlRichDescribeTable extends SqlCall {
     public static final SqlSpecialOperator OPERATOR =
             new SqlSpecialOperator("DESCRIBE TABLE", SqlKind.DESCRIBE_TABLE);
     protected final SqlIdentifier tableNameIdentifier;
-    private boolean isExtended = false;
+    private final boolean isExtended;
```

Comment:
Same, is this necessary?

Reply:
not strictly needed, but the IDE preferred it this way, so i wanted to leave the codebase cleaner than I found it